### PR TITLE
POWR-1178 update group node and my content to properly display reviewer

### PR DESCRIPTION
**Pull request by joshuami**

### CODE REVIEW

File: web/sites/default/config/views.view.media.yml [hunk 1]
```diff
@@ -4362,7 +4357,7 @@ display:
           exposed: true
           expose:
             operator_id: uid_op
-            label: Author
+            label: Name
```
Comment (kxwang) [inline web/sites/default/config/views.view.media.yml]: This line would revert the fix for "Author" filter name.

File: web/sites/default/config/views.view.media.yml [hunk 2]
```diff
@@ -7290,7 +7283,7 @@ display:
...
```
Comment (kxwang) [inline web/sites/default/config/views.view.media.yml]: Should be Author

File: web/sites/default/config/views.view.media.yml [hunk 3]
```diff
@@ -3479,7 +3479,7 @@ display:
...
```
Comment (kxwang) [inline web/sites/default/config/views.view.media.yml]: Change to Author

File: web/sites/default/config/views.view.media.yml (hunk 1 above)
Comment (joshuami) [reply web/sites/default/config/views.view.media.yml]: I must have missed the merge of your story into this branch. Fixed and rebuilding.

**Pull request merged**